.github/workflows/release-followups.yml

```diff
@@ -10,6 +10,7 @@ env:
   # Variables needed
   newPulsarVersion: "null" # Set by 'findCurrentPulsarVersion' script
   GH_ISSUE_TOKEN: ${{ secrets.AUTH_TOKEN_GITHUB_ISSUES }} # Set our secret auth token for `gh` CLI to use
+  GH_TOKEN: ${{ secrets.AUTH_TOKEN_GITHUB }} # Set our secret auth token for `gh` CLI to use
```

Comment:
We may not have to add this if no token is provided on the `actions/checkout` task? So it uses the default token, since the `pulsar-edit/pulsar` repo is public hopefully the default token is enough?

Reply:
Yeah just made that change, lets see if the implicit token will work. But you're right, we are just cloning a public repo, we shouldn't need any special permissions